js/ui/marker.js

```diff
@@ -44,6 +44,7 @@ Marker.prototype = {
         this._map = map;
         map.getCanvasContainer().appendChild(this._element);
         map.on('move', this._update);
+        map.on('moveend', this._update);
         this._update();
```

Comment:
We should round the coords upon marker initialization too. 

Perhaps we could do this by adding a `roundPos` option to `Marker#_update`
```js
map.on('move', this._update.bind(this));
map.on('moveend', this._update.bind(this, {roundPos: true}));
this._update({roundPos: true});
```